test/tokenServiceFullFlow.js

```diff
@@ -56,8 +56,13 @@ describe('Interchain Token Service Full Flow', () => {
         });
 
         it('Should register the token and initiate its deployment on other chains', async () => {
-            const tx1 = await service.populateTransaction.registerCanonicalToken(token.address);
```

Comment:
@milapsheth the issue is we can't register token and call other commands in the same multicall TX. The [token manager check for code length](https://github.com/axelarnetwork/interchain-token-service/blob/c618c91aa5855639bcfaaa5d7cd60c1a744d8702/contracts/interchain-token-service/InterchainTokenService.sol#L165) fails if in the same TX

Reply:
Are you sure? It's worked before even though the same check was in place.